setup.py

```diff
@@ -31,6 +31,10 @@
     keywords='pydocstyle, PEP 257, pep257, PEP 8, pep8, docstrings',
     packages=('pydocstyle',),
     package_dir={'': 'src'},
+    package_data={'pydocstyle': ['data/*.txt']},
+    install_requires=[
+        'snowballstemmer==1.2.1',
```

Comment:
It's okay to pin version in the development requirements, but the install requirements should be as loose as possible, so please remove the version number.